### PR TITLE
Small coverage improvements

### PR DESCRIPTION
**Pull request by Luthaf**
While checking what we do here to help setup coverage on metatensor-models, I found a couple of issues.

# Contributor (creator of pull-request) checklist

 - [ ] ~Tests updated (for new features and bugfixes)?~
 - [ ] ~Documentation updated (for new features)?~
 - [ ] ~Issue referenced (for PRs that solve an issue)?~

# Reviewer checklist

 - [ ] CHANGELOG updated with public API or any other important changes?

----
📚 Documentation preview 📚: https://metatensor--531.org.readthedocs.build/en/531/

### CODE REVIEW

**Comment by PicoCentauri** [comment]
Interesting that `combine` is not able to find the file for  'python/metatensor-learn/.coverage'. Can you maybe check if the file is created locally?

**Comment by Luthaf** [comment]
It was not created on CI, we where only running a subset of tests there.

File: .github/workflows/coverage.yml [hunk 1]
```diff
@@ -46,11 +46,7 @@ jobs:
           echo "CMAKE_CXX_COMPILER_LAUNCHER=sccache" >> $GITHUB_ENV
 
       - name: collect Python coverage
-        run: |
```
Comment (PicoCentauri) [inline .github/workflows/coverage.yml:49]: clever!

**Review by PicoCentauri — APPROVED** [review]

**Comment by Luthaf** [comment]
Coverage still seems to be missing for -learn: https://app.codecov.io/gh/lab-cosmo/metatensor/commit/74ddd5bc99029f5f22125ac20ed4f86214bd3f31/blob/python/metatensor-learn/metatensor/learn/data/collate.py

**Pull request merged**